### ISSUE TITLE
Learning Neural Networks

### ISSUE THREAD

**Issue by gmanc2** [anujdutt9/Disease-Prediction-from-Symptoms#6]
Hello, I apologize if this is a dumb question or if I'm not experienced enough to work on this. I was hoping to use this as one of my first neural network projects to enrich my knowledge of Python and machine algorithms I wanted to input data. For example, if I had 3 patients who had itching scratching and headaches and I wanted to input that data do your code to tell me what disease they most likely had I was wondering how I would accomplish that. I looked through your code and I understand most of it but I'm failing to understand if this is just a dataset or if it can be used to process user information for the desired outcome. Whether that be inputting symptoms and getting the most likely disease or input diseases and getting the most likely output. If I'm confused and if I'm misinterpreting the purpose of this please let me know! I really appreciate your work for the Neural networking and physician field to better assist those in need and I cannot thank you enough for this project!

**Comment by anujdutt9** [comment]
Hi @gmanc2, sure. You can take the existing dataset and train a simple MLP to begin with that takes in the features which are One Hot Encoded and predicts the output labels. Although, since there are a lot of labels/diseases/prognosis, I wouldn't be surprised if that model overfits a bit. But anyhow, that should be a good starting point.
